pstree.go

```diff
@@ -59,7 +59,7 @@ func New() (*Tree, error) {
 }
 
 const (
-	statfmt = "%d %s %c %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d"
+	statfmt = "%d %c %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d %d"
```

Comment:
we could perhaps drop the leading `%d` PID and only parse *after* the "comm" field.